pysatMissions/instruments/missions_sgp4.py

```diff
@@ -65,10 +68,10 @@ def init(self):
 clean = mcore._clean
 
 
-def load(fnames, tag=None, inst_id=None,
-         TLE1=None, TLE2=None, alt_periapsis=None, alt_apoapsis=None,
-         inclination=None, raan=None, arg_periapsis=None, mean_anomaly=None,
-         bstar=None, num_samples=None, cadence='1S'):
+def load(fnames, tag=None, inst_id=None, TLE1=None, TLE2=None,
+         alt_periapsis=None, alt_apoapsis=None,
+         inclination=None, raan=0., arg_periapsis=0., mean_anomaly=0.,
```

Comment:
I see that raan and arg_periapsis etc. are used based on kwarg settings. Seems possible a user may not know when they are used. I'd recommend adding info on the docstring to inclination that it is the key for the other parameters. Info for the other variables should be updated to reflect the new defaults.